Compilation issue when building

**Describe the bug**
Compilation error when building project using the package

**To Reproduce**
Steps to reproduce the behavior:
1. Create a new react-native project : `npx create-expo-app@latest`
2. Add android folder with expo prebuild : `npx expo prebuild --platform android`
3. Compile project : `npm run android`
4. Build Success
5. add package to project : `npm i @google/react-native-make-payment`
6. Compile project : `npm run android`
7. Build **FAILS**

**Expected behavior**
Build should success after installing package

**Desktop**
 - OS: Ubuntu 22.04

**Additional context**

Error log :

> Task :google_react-native-make-payment:compileDebugKotlin FAILED
e: file:///home/****/workspace/test-google-payment/node_modules/@google/react-native-make-payment/android/src/main/java/com/makepayment/MakePaymentModule.kt:99:32 Type mismatch: inferred type is Exception? but Throwable was expected

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':google_react-native-make-payment:compileDebugKotlin'.
> A failure occurred while executing org.jetbrains.kotlin.compilerRunner.GradleCompilerRunnerWithWorkers$GradleKotlinCompilerWorkAction

--- a/android/src/main/java/com/makepayment/MakePaymentModule.kt
+++ b/android/src/main/java/com/makepayment/MakePaymentModule.kt
@@ -48,9 +48,9 @@ class MakePaymentModule(reactContext: ReactApplicationContext) :
                         // Received response from Google Pay API - resolve the current promise.
                         Activity.RESULT_OK -> {
                             try {
-                                intent?.let { intent ->
+                                intent?.let { i ->
                                     val paymentData = Convert.jsonStringToMap(
-                                        PaymentData.getFromIntent(intent)!!.toJson()
+                                        PaymentData.getFromIntent(i)!!.toJson()
                                     )
                                     loadPaymentDataPromise.resolve(paymentData)
                                 }
@@ -96,7 +96,7 @@ class MakePaymentModule(reactContext: ReactApplicationContext) :
             if (completedTask.isSuccessful) {
                 promise.resolve(completedTask.result)
             } else {
-                promise.reject(completedTask.exception)
+                promise.reject(E_UNABLE_TO_DETERMINE_GOOGLE_PAY_READINESS, completedTask.exception)
             }
         }
     }
@@ -131,6 +131,7 @@ class MakePaymentModule(reactContext: ReactApplicationContext) :
     companion object {
         private const val NAME = "MakePayment"
         private const val LOAD_PAYMENT_DATA_REQUEST_CODE = 991
+        private const val E_UNABLE_TO_DETERMINE_GOOGLE_PAY_READINESS = "E_UNABLE_TO_DETERMINE_GOOGLE_PAY_READINESS"
 
         private fun createPaymentsClient(context: ReactApplicationContext): PaymentsClient {
             Log.d(NAME, "createPaymentsClient env=" + BuildConfig.GOOGLE_PAY_ENVIRONMENT)

